scipy/signal/filter_design.py

```diff
@@ -3653,6 +3653,232 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, rp=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    A notch filter is a band-stop filter with a narrow bandwidth
+    (high quality factor). It rejects a narrow frequency band and
+    leaves the rest of the spectrum little changed.
+
+    Parameters
+    ----------
+    w0 : float
+        Normalized frequency to remove from a signal. It is a
+        scalar that must satisfy  0 < w0 < 1, with w0 = 1 corresponding
+        to half of the sampling frequency.
+    bw : float
+        Normalized filter bandwidth (the diference between up and lower
+        passband edges).
+    rp : float, optional
```

Comment:
Nikolay I still think it is `rp`... Lets me explain my way of thinking and you check if I am wrong:

The figure bellows show the specifications of a band-stop filter:
![fullsizerender 2](https://cloud.githubusercontent.com/assets/16557411/17193152/11bd045a-5428-11e6-9f39-4f531f5a0c3c.jpg)


My interpretation is that the notch filter have an rejection band of a single point `w0` and an complete attenuation at that point `rs = infinity`

![fullsizerender 5](https://cloud.githubusercontent.com/assets/16557411/17193216/7524c316-5428-11e6-8c8e-1019f093448e.jpg)